go/exocom/src/exocom/exocom.go

```diff
@@ -106,3 +125,71 @@ func parseRegisterMessagePayload(message structs.Message) (string, error) {
 	}
 	return "", fmt.Errorf("Invalid register message payload: %v", message.Payload)
 }
+
+func (e *ExoCom) send(message structs.Message) error {
+	publicMessageName := messageTranslator.GetPublicMessageName(&messageTranslator.GetPublicMessageNameOptions{
+		Namespace:           e.clientRegistry.Clients[message.Sender].InternalNamespace,
+		ClientName:          message.Sender,
+		InternalMessageName: message.Name,
+	})
+	originalName := message.Name
+	message.Name = publicMessageName
```

Comment:
I think we can avoid overriding this as well. I'd like to try and not update message. I think we can get away with using a publicMessageName variable that gets passed into sendToServices